build.sbt

```diff
@@ -35,21 +36,26 @@ lazy val commonSettings = Seq(
   organization := "com.cognite.spark.datasource",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "3.18." + patchVersion,
+  version := "3.19." + patchVersion,
```

Comment:
is this the only real change in this file? Since we don't enforce the formatting would be easier to review without reformats

Reply:
ah yes it is... I must admit I got annoyed with permanentely being blocked by fmt from running the pipeline so I enabled autoformat at some point, I can disable and revert.